core/src/main/java/de/danoeh/antennapod/core/storage/DBWriter.java

```diff
@@ -244,6 +244,16 @@ public static Future<?> clearPlaybackHistory() {
         });
     }
 
+    public static Future<?> deleteFromPlaybackHistory(FeedItem feedItem) {
+        return dbExec.submit(() -> {
+            PodDBAdapter adapter = PodDBAdapter.getInstance();
+            adapter.open();
+            adapter.removeFromPlaybackHistory(feedItem);
```

Comment:
Instead of adding a new method, something like this should also work:

```
media.setPlaybackCompletionDate(new Date(0));
adapter.setFeedMediaPlaybackCompletionDate(media);
```